Failed to compile in gcc 4.9

Failed to compile in gcc 4.9

rt8192cu/os_dep/linux/usb_intf.c:1580:52: error: macro "\_\_DATE\_\_" might prevent reproducible builds [-Werror=date-time]

rt8192cu/os_dep/linux/usb_intf.c:1580:1: error: macro "\_\_TIME\_\_" might prevent reproducible builds [-Werror=date-time]

--- a/os_dep/linux/usb_intf.c
+++ b/os_dep/linux/usb_intf.c
@@ -1577,7 +1577,6 @@ static int __init rtw_drv_entry(void)
 	RT_TRACE(_module_hci_intfs_c_,_drv_err_,("+rtw_drv_entry\n"));
 
 	DBG_871X(DRV_NAME " driver version=%s\n", DRIVERVERSION);
-	DBG_871X("build time: %s %s\n", __DATE__, __TIME__);
 
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,24))
 	//console_suspend_enabled=0;

